optimum/exporters/tasks.py

```diff
@@ -2063,10 +2070,13 @@ def get_model_from_task(
                     model_class_name = config.architectures[0]
 
```

Comment:
Could make sense to have function that works for all supported libraries (diffusers / timm / sentence-transformers), taking instead the `library_name` and doing a check with `is_xxxxx_available`, so that we can call this function and do this check for all libraries  anytime needed